### PR TITLE
vampiro

### PR DESCRIPTION
**Pull request by RDLL**
One of the most traditional cocktails in Mexico

### CODE REVIEW

File: src/recipes/vampire.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "2 slices of natural orange"
+    ],
+    "directions": [
+        "Add the ice cubes and Don Julio tequila (one ounce) in each glass.",
+        "Then add the siracha sauce to the tomato juice and stir.",
+        "This cocktail is prepared in layers, so start with the first layer by pouring a little of the tomato juice, then add the grapefruit or grapefruit juice and, finally, the orange juice.",
+        "Garnish with a slice of pink grapefruit and a slice of natural orange.",
+        "Drink this vampire cocktail slowly and enjoy while you drip with fresh Mexican sauce and corn chips before the strong meal."
+    ],
+    "image": "Vampiro.jpg"
```
Comment (alfg) [inline src/recipes/vampire.json]: Can we update the casing to all lower. `vampiro.jpg` and the filename.

Thanks!

Comment (RDLL) [reply src/recipes/vampire.json]: of course, no problem

Comment (RDLL) [reply src/recipes/vampire.json]: all fixed

**Comment by alfg** [comment]
Perfect! Thanks!

**Pull request merged**